packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -841,6 +844,9 @@ class PageView extends StatefulWidget {
   final ScrollPhysics? physics;
 
   /// Set to false to disable page snapping, useful for custom scroll behavior.
+  ///
+  /// If the [padEnds] is false, the page will snap to the beginning of the
```

Comment:
Sorry I realized i missed something
```
/// If the [padEnds] is false and [PageController.viewportFraction] < 1.0, the page will snap to the beginning of the viewport; otherwise, the page will snap to the center of the viewport